GeneralConsoleGame/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GeneralConsoleGame
```

Comment:
This project is obsolete now. You have project that has this one's functionality. you can delete it from repository. :)

Reply:
Done. Thank you.